packages/expect/src/jasmineUtils.ts

```diff
@@ -108,7 +108,7 @@ function eq(
     case '[object Number]':
       // `NaN`s are equivalent, but non-reflexive. An `egal` comparison is performed for
```

Comment:
Comment obsolete now?

Reply:
Indeed ;)

I should maybe add another comment related to the fact that we are doing `Object.is(Number(a), Number(b))` and not `Object.is(a, b)`.

I was thinking of something like (the comment of String adapted):
> Primitives and their corresponding object wrappers are equivalent; thus, `1` is
> equivalent to `new Number(1)`.

Depending on your preferences I can change `Object.is(Number(a), Number(b))` into `Object.is(+a, +b)` which is equivalent.

Reply:
IMO doesn't really need any explanation, and `Number` is fine :)